src/components/OptionsList.js

```diff
@@ -0,0 +1,127 @@
+import _ from 'underscore';
+import React from 'react';
+import {View, SectionList} from 'react-native';
+import PropTypes from 'prop-types';
+import styles from '../styles/styles';
+import KeyboardSpacer from './KeyboardSpacer';
+import ChatLinkRow from '../pages/home/sidebar/ChatLinkRow';
+
+const propTypes = {
+    /** Extra styles for the section list container */
```

Comment:
the least nab I've ever written: I'm used to the `propType` comments started with `//`, any particular reason for the change here?

Reply:
haha yeah I am proposing we migrate to this style and added it to the style guide here:

https://github.com/Expensify/Expensify.cash/blob/master/STYLE.md#method-naming-and-code-documentation